docs/community/libraries.md

```diff
@@ -0,0 +1,7 @@
+# Libraries
+
+Take a look at some of the libraries that our community has built. Edit this page on GitHub to list your libraries or plugins.
+
+- [fluxible-action-utils](https://github.com/yahoo/fluxible-action-utils) - utility methods to aid in writing [actions](http://fluxible.io/api/fluxible-context.html#executeaction-action-payload-callback-) for [fluxible](http://fluxible.io) based applications.
+- [fluxible-immutable-store](https://www.npmjs.com/packages/fluxible-immutable-store) - A helper for creating Fluxible/Dispatchr BaseStores that manage immutable data structures.
+- [fumble](https://github.com/yahoo/fumble) - Simple error objects in node. Created specifically to be used with the [fetchr](https://github.com/yahoo/fetchr) library and based on [hapi.js](http://hapijs.com/)' [Boom](https://github.com/hapijs/boom).
```

Comment:
Add link to `fluxible-immutable-utils`.

- [fluxible-immutable-utils](https://github.com/yahoo/fluxible-immutable-utils) - A mixin that provides a convenient interface for using Immutable.js inside react components.

We should also add a space before each bullet point.